build.py

```diff
@@ -95,6 +95,7 @@ def validate_cuda_home(cuda_home: str | bytes | os.PathLike | None):
 
 def build(
     skip_wheel: bool = False,
+    skip_python: bool = False,
```

Comment:
What is this solving? Should probably be a separate change if needed.

Reply:
reverted these changes for this PR. I found it useful to make optional though.